### ISSUE TITLE
Bitrate, Resolution, Latency?

### ISSUE THREAD

**Issue by TheCutter** [psolyca/Yi_4k_ROOTFS#8]
Hey,

first thank you very much for the custom firmware.
I tried to get my Yi 4k running as a rtsp device to use it as a webcam. I got it working with Ethernet over USB on windows with Event Manager.

But:
1. What is the resolution and bitrate that is used, when started via Event Manager?
2. Is it possible to reduce the latency of the stream to use it as a webcam? Currently the latency is around 2 seconds.

Thanks in advance

**Comment by psolyca** [comment]
For the resolution, it should be the default stream resolution... do not remember which one ^^;
We already tried to reduce latency when using the RTSP server and OBS but never really managed to get it low. We do not know why this latency is so high.
Currently, i'm on another subject for the Yi4k so I will look at stream trouble later.

**Comment by TheCutter** [comment]
Thank you! 
I read somewhere that it has something to do with the internal handling of the stream.

**Comment by psolyca** [comment]
Yep, and there are 2 different stream, one created for the ViewFinder (RTSP) and one when streaming (RTMP) but they have both high latency.
I do not know if I can change something without the source code.

**Comment by TheCutter** [comment]
Ah okay I see. So the internal ViewFinder Display is also a stream?
But on the screen it is played in realtime. Thats strange.

**Comment by psolyca** [comment]
Yes but you can find it on the cam using the official app when viewing in realtime. It's the RTSP stream.
RTMP is started when starting a stream to a service.
